src/app/(sidebar)/transactions-explorer/tx/[tx]/components/TransactionDetails.tsx

```diff
@@ -0,0 +1,252 @@
+        greater the required fee.
+      </Tooltip>
+    </Box>
+  );
+
+  const feeChargedElement = (
+    <Box align="center" direction="row" gap="xs">
+      Fee charged
+      <Tooltip
+        triggerEl={
+          <div className="Label__infoButton" role="button">
+            <Icon.InfoCircle />
+          </div>
+        }
+      >
+        Actually charged fee which can be lower than the fee specified in the
+        transaction. Each transaction sets a fee that is paid by the source
+        account. The more operations in the transaction, the greater the
+        required fee.
+      </Tooltip>
+    </Box>
+  );
+
+  const goToAccount: MouseEventHandler = (event) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    endpoints.updateParams({ account_id: sourceAccount });
+
+    delayedAction({
+      action: () => {
+        router.push(Routes.ENDPOINTS_ACCOUNTS_SINGLE);
+      },
+      delay: 100,
+    });
+  };
+
+  const goToTransaction: MouseEventHandler = (event) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    endpoints.updateParams({ transaction: tx.txHash });
+
+    delayedAction({
+      action: () => {
+        router.push(Routes.ENDPOINTS_TRANSACTIONS_SINGLE);
+      },
+      delay: 100,
+    });
+  };
+
+  useIsXdrInit();
+
+  useEffect(() => {
+    const parse = async () => {
+      try {
+        const envelopeXdr = tx.envelopeXdr.toXDR().toString("base64");
+        const guesses = StellarXdr.guess(envelopeXdr);
+        const json = jsonParse(
+          StellarXdr.decode(guesses[0], envelopeXdr),
+          null,
+          parseNumberAndBigInt,
+        );
+        setXdrJson(json as Record<string, unknown>);
+        setXdr(envelopeXdr);
+
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      } catch (e) {
+        // do nothing
+      }
+    };
+
+    parse();
+  }, [tx]);
+
+  return (
+    <Box gap="md" data-testid="explorer">
+      <PageCard heading="Transaction Envelope">
+        <Box gap="xs" align="start">
+          <InfoField
+            label="Status"
+            value={
+              <Box
+                direction="row"
+                align="center"
+                gap="xs"
+                addlClassName={tx.status.toLowerCase()}
+              >
+                <Text size="xs" as="span">
+                  {success ? "Successful" : "Failed"}
+                </Text>
+                {success ? (
+                  <Icon.CheckCircle size="xs" color="currentColor" />
+                ) : (
+                  <Icon.XCircle size="xs" color="currentColor" />
+                )}
+              </Box>
+            }
+          />
+          <InfoField
+            label="Transaction"
+            value={
+              <SdsLink
+                onClick={goToTransaction}
+                href={buildEndpointHref(Routes.ENDPOINTS_TRANSACTIONS_SINGLE, {
+                  transaction: tx.txHash,
+                })}
```

Comment:
Do we need `href` here if we're doing `onClick` action?

Reply:
Yes, so you can open it on a new tab. I've been doing it frequently.